src/Transform/ONNX/SimplifyShapeRelatedOps.cpp

```diff
@@ -517,7 +518,9 @@ void SimplifyShapeRelatedOpsPass::runOnOperation() {
   for (unsigned i = 0; i < 3; ++i) {
     topDownShapeSimplification(context, moduleOp);
     OpPassManager pm("builtin.module");
+    if ((/*enableConstantProp*/ onnx_mlir::OptimizationLevel >= 3 && !onnx_mlir::enableConstantProp) || onnx_mlir::enableConstantProp) {
```

Comment:
simplify the logic as mentioned before, and add a local parameter optLevel that is set when constructing this object.

Reply:
I can remove `onnx_mlir::OptimizationLevel` and use `optLevel` instead... I will use the enum like in NNPA.